cumulusci/tasks/metadata_etl/base.py

```diff
@@ -305,3 +306,48 @@ def _transform(self):
                 removed_api_names.add(api_name)
 
         self.api_names = self.api_names - removed_api_names
+
+
+class UpdateMetadataFirstChildTextTask(MetadataSingleEntityTransformTask):
+    """
+    Updates the metadata's first child tag's text.
+    Does not support tags with complex/object-like values.
+    """
+
+    task_options = {
+        "entity": {"description": "Metadata Entity", "required": True},
```

Comment:
Entities are called Metadata Types in the Metadata API docs; is there a reason to depart from that here?

Reply:
`metadata_etl` uses the `entity` variable, but yes!  I much prefer using the usual terminology

Reply:
updated